cdap-proto/src/main/java/io/cdap/cdap/proto/PreferencesDetail.java

```diff
@@ -105,5 +96,9 @@ public String toString() {
       "resolved='" + resolved +
       '}';
   }
+
+  public static Long getMaxSeqId(Long left, Long right) {
```

Comment:
there isn't any value in this method, can just call Long.max directly

Reply:
Done.